tests/dbapi20.py

```diff
@@ -191,8 +192,8 @@ def test_Exceptions(self):
             self.failUnless(issubclass(self.driver.Warning,Exception))
             self.failUnless(issubclass(self.driver.Error,Exception))
         else:
-            self.failUnless(issubclass(self.driver.Warning,StandardError))
-            self.failUnless(issubclass(self.driver.Error,StandardError))
+            self.failUnless(issubclass(self.driver.Warning,StandardError))  # type: ignore[name-defined]
```

Comment:
My intention was to not fix this file, which is a sort of historical drop-in. But if we have to work on it maybe it's better to do it properly, in case mypy complains? The Python 2 branch might be probably just removed, and I'm thinking about the deprecated method names.

Reply:
I wasn't sure about this file, so went on with the least invasive changes for now.